packages/kernel-6.12/config-bottlerocket

```diff
@@ -108,6 +108,9 @@ CONFIG_SECURITY_LOCKDOWN_LSM_EARLY=y
 # Enable landlock
 CONFIG_SECURITY_LANDLOCK=y
 
+# Add landlock to the list of LSMs.
+CONFIG_LSM="landlock,lockdown,yama,loadpin,safesetid,selinux,smack,tomoyo,apparmor,ipe,bpf"
```

Comment:
This is the upstream default for kernel 6.12: [Kconfig](https://elixir.bootlin.com/linux/v6.12.39/source/security/Kconfig#L271).